pre-document.sh

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
Instead of removing this file because it’s not needed in major, I left it so we can reuse it again if we need to for CanJS 7.